#Number of Controllers
int32 num_ctrls

# Type of dominant controller: force or moment
string domType

# Desired force/moment (3D)
geometry_msgs/Vector3[] domDes

# Gains force/moment (3D) this is optional if not given use default
geometry_msgs/Vector3[] domGains

# Type of subordianTE controller: force or moment
string subType

# Desired force/moment (3D)
geometry_msgs/Vector3[] subDes

# Gains force/moment (3D) this is optional if not given use default
geometry_msgs/Vector3[] subGains

---
# return
int32 output
